modules/data_release/templates/menu_data_release.tpl

```diff
@@ -84,6 +95,61 @@
     </div>
 </div>
 
+<button type="button" name = "managepermissions" class = "btn btn-sm btn-primary" data-toggle="modal" data-target="#managePermissionsModal">Manage Permissions</button>
+
+<div class="modal fade" id="managePermissionsModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
+                <h3 class="modal-title" id="myModalLabel">Manage Permissions</h3>
+            </div>
+            <form name = "uploadForm" id = "uploadForm" method = "POST" enctype="multipart/form-data" action="{$baseurl}/data_release/ajax/AddPermission.php">
```

Comment:
This is the third form with id=uploadForm
`[DOM] Found 3 elements with non-unique id #uploadForm`